No artifacts found on ubuntu-latest

```
    Finished release [optimized] target(s) in 3m 46s
    Bundling diffuse_3.0.0-alpha.1_amd64.deb
    Bundling diffuse.AppImage
    Finished 2 bundles at:
        /home/runner/work/diffuse/diffuse/src-tauri/target/release/bundle/deb/diffuse_3.0.0-alpha.1_amd64.deb
        /home/runner/work/diffuse/diffuse/src-tauri/target/release/bundle/appimage/diffuse.AppImage

 app:tauri Shutting down tauri process... +1ms
##[error]No artifacts were found.
```

_Workflow: https://gist.github.com/icidasset/52c289a08f4a3cd3262a43ca56ce4ae2_

It does pass on `macos-latest` though, although no artifacts were uploaded to the release 🤔 
Fails on `windows-latest`, see  #40

Thanks!

--- a/README.md
+++ b/README.md
@@ -102,7 +102,7 @@ jobs:
   create-release:
     runs-on: ubuntu-latest
     outputs:
-      RELEASE_UPLOAD_ID: ${{ steps.create_release.outputs.upload_id }}
+      RELEASE_UPLOAD_ID: ${{ steps.create_release.outputs.id }}
 
     steps:
       - uses: actions/checkout@v2

